src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionSettings.cs

```diff
@@ -123,6 +121,8 @@ public HttpConnectionSettings CloneAndNormalize()
                 _plaintextStreamFilter = _plaintextStreamFilter,
                 _initialHttp2StreamWindowSize = _initialHttp2StreamWindowSize,
                 _activityHeadersPropagator = _activityHeadersPropagator,
+                _defaultCredentialsUsedForProxy = _proxy != null && (_proxy.Credentials == CredentialCache.DefaultCredentials || _defaultProxyCredentials == CredentialCache.DefaultCredentials),
```

Comment:
Should we remove the place where these are set in the constructor? Seems like that code doesn't do anything, and worse, is confusing.

Reply:
I can run some tests to see if that still works without the constructor changes.  Or eat least add some comments.

Reply:
I can move it. I felt it may not be worth do do that for a single test but perhaps it can grow over time.

Reply:
done.